src/zjs_aio.c

```diff
@@ -302,10 +301,25 @@ static ZJS_DECL_FUNC(zjs_aio_open)
     jerry_set_prototype(pinobj, zjs_aio_prototype);
     zjs_obj_add_number(pinobj, "pin", pin);
 
+    aio_handle_t *handle = zjs_aio_alloc_handle();
+    if (!handle)
+        return zjs_error("could not allocate handle");
+
+      jerry_set_object_native_pointer(pinobj, handle, &aio_type_info);
+
     return pinobj;
 }
 
-jerry_value_t zjs_aio_init()
+void zjs_aio_cleanup()
```

Comment:
These cleanup functions can now be made static.

Reply:
Good point, updating...